docs/release-notes/release-notes-0.15.1.md

```diff
@@ -10,6 +10,11 @@
 * [Add minor comment](https://github.com/lightningnetwork/lnd/pull/6559) on
   subscribe/cancel/lookup invoice parameter encoding.
 
+## Bug Fixes
+
+* [Update urfave](https://github.com/lightningnetwork/lnd/pull/6559) because
+  of a flag parsing bug].
```

Comment:
Broken formatting. Also (nit), "urfave" is the username, the library that's being updated is called `cli`.